server/routes/chat_routes.ts

```diff
@@ -123,16 +146,20 @@ const feedbackRoute = {
     body: schema.object({
       satisfaction: schema.boolean(),
     }),
+    query: schema.object({
+      dataSourceId: schema.maybe(schema.string()),
```

Comment:
Shall we add some UT for dataSourceId query passed?

Reply:
We could add UT for data source specifically, for now this PR is depended by other PRs, right?

Reply:
Do you mean will be added in the following PRs?

Reply:
I remember that you wanted this PR being merged ASAP, if not, we could try to add tests in this PR to make this fully.

Reply:
I think you can added them first. At least we need to verify the route handler called getOpenSearchClientTransport with passed data source id.